src/views/NoteEditView.vue

```diff
@@ -1,7 +1,366 @@
+    </div>
+    <div class="editor-container">
+      <div class="editor-pane">
+        <div v-if="notesStore.loading" class="loading-overlay">
+          <div class="loading-message">読み込み中...</div>
+        </div>
+        <div class="editor-content">
+          <textarea
+            placeholder="このノートは、Markdown 形式で入力できます。traQ のチャンネルと紐づけることで、ノートを簡単に管理できます。"
+            v-model="noteBody"
+            :disabled="notesStore.loading"
+          ></textarea>
+        </div>
+      </div>
+      <div class="preview-pane">
+        <div class="preview-content">
+          <h1>Lorem ipsum</h1>
+          <p>
+            dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore
+            et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+            laboris nisi ut aliquip ex ea commodo consequat.
+          </p>
+
+          <h2>サブヘッダー</h2>
+
+          <p><strong>太字のテキスト</strong>と<em>斜体のテキスト</em>の例です。</p>
+
+          <ul>
+            <li>リストアイテム1</li>
+            <li>リストアイテム2</li>
+            <li>リストアイテム3</li>
+          </ul>
+
+          <ol>
+            <li>番号付きリスト1</li>
+            <li>番号付きリスト2</li>
+            <li>番号付きリスト3</li>
+          </ol>
+
+          <p><code>インラインコード</code>の例です。</p>
+
+          <pre><code class="language-javascript">// コードブロックの例
+function hello() {
+  console.log("Hello, World!");
+}</code></pre>
+
+          <blockquote>
+            <p>
+              これは引用文です。<br />
+              複数行にわたって書くことができます。
+            </p>
+          </blockquote>
+
+          <p><a href="https://example.com">リンクの例</a></p>
+        </div>
+      </div>
+    </div>
+  </div>
 </template>
 
 <script setup lang="ts">
+// 必要なものをインポートする
+import { onMounted, ref, watch } from 'vue';
+import { useRoute } from 'vue-router';
+import { useNotesStore } from '@/stores/notes';
+
+const route = useRoute();
+const notesStore = useNotesStore();
+
+/**
+ * 編集ページのパス `/notes/:noteId/edit` にある `:noteId` を取得する
+ * 現時点では `uuid-1` と `uuid-2`
+ */
+const noteId = route.params.noteId as string;
+console.log(noteId);
+
+/**
+ * ノートの本文
+ */
+const noteBody = ref('');
 
+/**
+ * ノートが紐づけられたチャンネル
+ */
+const noteChannel = ref('');
+
+/**
+ * ノートに付与されたタグ
+ */
+const noteTags = ref('');
+
+// currentNoteが変更されたときに、リアクティブ変数を更新する
+watch(
+  () => notesStore.currentNote,
+  (newNote) => {
+    if (newNote) {
+      noteBody.value = newNote.body || '';
+      noteChannel.value = newNote.channel || '';
+      // タグの実装は後回し
+      noteTags.value = '';
+
+      // ちゃんと取得できているか確認
+      console.log('noteBody updated:', noteBody.value);
+      console.log('noteChannel updated:', noteChannel.value);
+      console.log('noteTags updated:', noteTags.value);
+    }
+  },
+  { immediate: true }
+);
+
+// コンポーネントがマウントされたら、ノートを取得する
+onMounted(async () => {
+  if (noteId) {
+    // ノート一覧とノート詳細の両方を取得
+    await Promise.all([notesStore.fetchNotes(), notesStore.fetchNoteById(noteId)]);
+  }
+});
 </script>
+
```

Comment:
以下のような機構を入れるべきです
`src/stores/notes.ts` に `clearCurrentNote` などがあるべきなんですが実装してないので、良い感じのところにこれをねじ込んでほしいです

```ts name=src/stores/notes.ts
function clearCurrentNote() {
  currentNote.value = null;
  error.value = null;
}
```

```suggestion
onUnmounted(() => {
  // currentNote の情報をクリアする関数
});

</script>

```